packages/sanity/src/core/form/inputs/ReferenceInput/ReferenceField.tsx

```diff
@@ -273,7 +281,7 @@ export function ReferenceField(props: ReferenceFieldProps) {
             <Flex gap={1} align="center">
               <ReferencePreviewCard
                 flex={1}
-                forwardedAs={EditReferenceLink as any}
+                forwardedAs={forwardedAs}
                 tone="inherit"
                 radius={2}
                 data-as="a"
```

Comment:
```suggestion
                data-as={refType?.name ? 'a' : undefined}
```

What do you think about the above? Conditionally setting `data-as` will mean that the preview card won't show a hand cursor prior to the reference type being known

Example video below (red = no ref type, green = ref type loaded)

https://user-images.githubusercontent.com/209129/221065866-24659d16-2ba1-4ccc-b2df-e19e2fa24733.mp4

_Ideally_ the link wouldn't be clickable until the full preview is loaded in its entirety, but that's probably a story for another time!

Reply:
Good point, will add that 👍